### PR TITLE
added tooltip

### PR DESCRIPTION
**Pull request by subhamkrai**
Signed-off-by: subhamkrai <subhamkumarrai03@gmail.com>

### CODE REVIEW

**Comment by leecalcote** [comment]
@subhamkrai nice job on including your git commit --sign-off.

File: assets/css/materialize.css [hunk 1]
```diff
@@ -2897,6 +2898,40 @@ ul.staggered-list li {
   list-style-type: none;
 
 }
+.comparison {
+          float:left; display: inline;
```
Comment (leecalcote) [inline assets/css/materialize.css]: Good work, but no need for indentation here.

File: _includes/partials/hori-social.html [hunk 2]
```diff
@@ -1,16 +1,34 @@
...
+      
+      <div class="tooltip">
+      <a  href="{{site.youtube}}">
+        <img class="small-on-med-and-down"width="40px" src="/assets/images/youtube.svg" />
+        <span class="tooltiptext">Watch community meeting recordings</span></a></div>
+      
+        <div class="tooltip">
+      <a  href="{{site.docker}}">
+        <img class="small-on-med-and-down devicon-docker-plain" width="40px" src="/assets/images/docker.svg" />
+        <span class="tooltiptext">Access Docker images for Layer5 projects</span></a></div>
```
Comment (leecalcote) [inline _includes/partials/hori-social.html]: Docker was accidentally misspelled as "Doker". I corrected this and updated to include a call to action for the user.

Comment (subhamkrai) [reply _includes/partials/hori-social.html]: Sorry for the typo.

File: assets/css/materialize.css (hunk 1 above)
Comment (subhamkrai) [reply assets/css/materialize.css]: Okay

Comment (leecalcote) [reply assets/css/materialize.css]: @subhamkrai looking good! Your changes have been deployed to the site. Looks nice.

Comment (subhamkrai) [reply assets/css/materialize.css]: @leecalcote thanks

**Pull request merged**